aws/table_aws_s3_multi_region_access_point.go

```diff
@@ -0,0 +1,269 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/s3control"
+	"github.com/aws/aws-sdk-go-v2/service/s3control/types"
+
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsS3MultiRegionAccessPoint(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_s3_multi_region_access_point",
+		Description: "AWS S3 Multi Region Access Point",
+		List: &plugin.ListConfig{
+			Hydrate: listS3MultiRegionAccessPoints,
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"InvalidParameter", "InvalidRequest"}),
+			},
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "account_id", Require: plugin.Optional},
+			},
+		},
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.AllColumns([]string{"name", "account_id"}),
+			Hydrate:    getS3MultiRegionAccessPoint,
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"NoSuchMultiRegionAccessPoint", "InvalidParameter", "InvalidRequest"}),
+			},
+		},
+		Columns: awsRegionalColumns([]*plugin.Column{
```

Comment:
Should this table really be regional if we're always connecting to us-west-2? Do we end up with duplicate results if you have multiple regions defined for your connection?

Reply:
AWS treats S3 multi-region access point as global resource, but the endpoint should be `us-west-2` for listing out the resources. We are getting the duplicate results if you have multiple regions defined for your connection.